src/renderer/components/wallet/NewMnemonicConfirm.tsx

```diff
@@ -199,7 +179,7 @@ const MnemonicConfirmScreen: React.FC<{ mnemonic: string; onConfirm: Function }>
         </Form.Item>
         <Form.Item>
           <Button size="large" type="primary" htmlType="submit" block>
-            {loadingMsg || 'Confirm'}
+            {loadingMsg || intl.formatMessage({ id: 'common.confirm' })}
```

Comment:
^ Similar to https://github.com/thorchain/asgardex-electron/pull/285/files#r457281840